core/src/main/kotlin/materialui/components/treeItem/treeItem.kt

```diff
@@ -0,0 +1,37 @@
+package materialui.components.treeItem
+
+import kotlinx.html.DIV
+import kotlinx.html.RP
+import kotlinx.html.Tag
+import kotlinx.html.TagConsumer
+import materialui.components.StandardProps
+import materialui.components.toolbar.enums.ToolbarStyle
+import org.w3c.dom.events.Event
+import react.*
+
+@JsModule("@material-ui/lab/TreeItem")
+private external val treeItemModule: dynamic
+
+external interface TreeItemProps : StandardProps {
+    var collapseIcon: ReactElement?
+    var endIcon: ReactElement?
+    var expandIcon: ReactElement?
+    var icon: ReactElement?
+    var label: ReactElement
+    var nodeId: String
+    var TransitionComponent: RComponent<RProps, RState>?
```

Comment:
Change `dynamic`, like this.

https://github.com/subroh0508/kotlin-material-ui/blob/89608ee06bfca2f0da2b37d61b55278be42f3b3c/core/src/main/kotlin/materialui/components/dialog/dialog.kt#L19

Reply:
And please set to all properties nullable.